### ISSUE TITLE
Suggestion: add a cheatsheet for functions available in ImageMath

### ISSUE THREAD

**Issue by mconsidine** [melix/astro4j#59]
Not entirely sure where ImageMath syntax/module is coming from, but a "cheat sheet" for functions/syntax/reserved words that can be used, ala what is available for OpenSCAD
  http://openscad.org/documentation.html#cheat-sheet
would be very helpful.

**Comment by melix** [comment]
I am currently working on a syntax highlighter:

![image](https://github.com/melix/astro4j/assets/316357/d4fed0e3-3b4e-4092-afb2-c71fb4052b4a)

But you also have the list of available functions [in the docs](https://melix.github.io/astro4j/latest/en/jsolex.html#_functions_available_in_imagemath).

**Comment by melix** [comment]
To answer the question, the ImageMath module is entirely coming from my brain :) It is inspired from PixelMath in PixInsight, with the idea of having a _simple_ dedicated language which is a bridge to the functions I have implemented in JSol'Ex.

I am also thinking of CLAHE but ddn't get to implementing it yet. As for plugins, I also have ideas about how to make that possible, but I will only do this once I have stabilized code a bit.

**Comment by mconsidine** [comment]
Understood on all points.  Thanks.On Jul 1, 2023 7:16 PM, Cédric Champeau ***@***.***> wrote:
To answer the question, the ImageMath module is entirely coming from my brain :) It is inspired from PixelMath in PixInsight, with the idea of having a simple dedicated language which is a bridge to the functions I have implemented in JSol'Ex.
I am also thinking of CLAHE but ddn't get to implementing it yet. As for plugins, I also have ideas about how to make that possible, but I will only do this once I have stabilized code a bit.
